proto/internal/temporal/server/api/historyservice/v1/service.proto

```diff
@@ -284,6 +284,32 @@ service HistoryService {
     rpc UpdateWorkflowExecution(UpdateWorkflowExecutionRequest) returns (UpdateWorkflowExecutionResponse) {
     }
 
+    // (-- api-linter: core::0131::synonyms=disabled
+    //     aip.dev/not-precedent:  --)
```

Comment:
I don't think we should ignore these linter warnings just because the other methods did. We should follow the resource-oriented AIP conventions. For example, this is a paginated list method against the HistoryEvent resource, so it should follow these guidelines: https://google.aip.dev/132

Reply:
rule `core::0131::synonyms` is complaining that our rpc name starts with `Read` and it likes `Get` instead. I'd argue there's much more value in keeping the full function name identical to the underlying operation naming than to conform to the resource-oriented API conventions here.

Reply:
Resolved, see: https://github.com/temporalio/temporal/pull/4198/files#r1178381497